code/modules/games/cards.dm

```diff
@@ -416,6 +416,7 @@ var/global/list/card_decks = list()
 /*** A special thing that steals a card from a deck, probably lost in maint somewhere. ***/
 /obj/item/hand/missing_card
 	name = "missing playing card"
+	abstract_type = /obj/item/hand/missing_card //Can't spawn this for tests because it kills itself
```

Comment:
You just told me not to add mildly related changes though? Wouldn't fixing fundamental issues with the card game be part of that? Because looking at the code and what depends on that behaviour, it seems like its not gonna be a simple change.

Also, there's no test exclusion list anymore, so that's the only way I can exclude buggy things from testing. Plus, I'm not actually sure this item is even meant to really exist in the world or if its just some pseudo item for pulling off some hack for the card game visuals. So I thought classifying it as abstract made sense.

Reply:
Like, its not just in init that update_icon is called. Its called in the base class, and its called in several other procs of the card stuff. Its probably going to need some code diving and debugging for properly changing that and not breaking anything.

Reply:
Sold! It doesn't seem much better to me, but I'll mark it as unspawnable. 

I just think the rabbit hole is deep enough as it is, without trying to fix every single problems along the way and burning out. I need to focus on one thing at a time. And I frankly don't want to have to figure out how the card games work, or if they even still work, in order to fix that.